plasma_framework/contracts/src/exits/OnlyFromExitGameController.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.5.0;
```

Comment:
Instead of creating an abstract contract with just a modifier and one field and call them respectively `OnlyFrom...`, do not we want to create a one abstract contract `OnlyFrom` which would not have any fields, but just a single modifier that would take a parameter: 
``` modifier onlyFrom(address caller){...}``` 

In this case we can just call `onlyFrom(exitGameController)` and there is so much less code boilerplate